### PR TITLE
Total num captures != London captures only, explaining enrich

### PR DESCRIPTION
**Pull request by 13072020**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ This API can be used to convert British National Grid eastings and northings to
 To complete the challenge you will need to:
 
 - Write a script called `enrich_data.py` to pull data from the API and enrich the data in the database with the latitudes and longitudes of the captures.
+    - it may take several minutes to receive the data from the API
```
Comment (keattang) [inline README.md:38]: Keen to omit this as I'd like to see how people work through this. Really great responses will realise this and may try parallelise. We can talk people through it on the day if we need to

File: README.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ This API can be used to convert British National Grid eastings and northings to
...
+    - what 'enrich' means is up to you - you might insert the values into new columns in the table, or perhaps create a new table with the values, or something else
```
Comment (keattang) [inline README.md:39]: Can we instead just add `Where you put the data in the database is up to you.` to line 37? Keen to hear how people talk through this and not bias their thinking

Comment (13072020) [reply README.md]: Sure, it occurred to me that people would prob just grab onto the first suggestion.

File: README.md (hunk 1 above)
Comment (13072020) [reply README.md]: Makes sense, my intent was to confirm that the API does indeed behave that way. But yes, can be vocal

**Review by keattang — APPROVED** [review]

**Pull request merged**